### PR TITLE
feat : Add grabCssPropertyFrom method for Nightmare

### LINKED ISSUE #642: how to check iframe height in nightmare
#### What are you trying to achieve?
verify iframe height
#### What do you get instead?
I.grabCssPropertyFrom is not function
> Provide console output if related. Use `--verbose` mode for more details.
I.grabCssPropertyFrom('#make-taller', 'height');
```bash
# paste output here
```

> Provide test source code if related

```php
// paste test
```

### Details

* CodeceptJS version:
* NodeJS Version:
* Operating System:
* Protractor || WebDriverIO || Nightmare version (if related)
* Configuration file:

```js
# paste suite config here
```

### PR DESCRIPTION
**Pull request by koushikmohan1996**
## Motivation/Description of the PR
Added grabCssPropertyFrom support for Nightmare helper.

Applicable helpers:

- [ ] Webdriver
- [ ] Puppeteer
- [X] Nightmare
- [ ] REST
- [ ] Appium
- [ ] Protractor
- [ ] TestCafe

fixes #642

## Type of change

- [ ] Breaking changes
- [X] New functionality
- [ ] Bug fix
- [ ] Documentation changes/updates
- [ ] Hot fix
- [ ] Markdown files fix - not related to source code

## Checklist:

- [x] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [ ] Lint checking (Run `npm run lint`)
- [x] Local tests are passed (Run `npm test`)

### CODE REVIEW

File: lib/helper/Nightmare.js [hunk 1]
```diff
@@ -835,6 +836,31 @@ class Nightmare extends Helper {
...
+      return (await this.browser.evaluate((el) => {
+        return window.getComputedStyle(window.codeceptjs.fetchElement(el));
+      }, el))[toCamelCase(prop)];
+    };
+
+    for (let index = 0; index < els.length; index++) {
+      const el = els[index];
+      assertElementExists(el, locator);
+      const cssValue = await getCssPropForElement(el, cssProperty);
+      array.push(cssValue);
```
Comment (PeterNgTr) [inline lib/helper/Nightmare.js]: may we try using `forEach`?

Comment (koushikmohan1996) [reply lib/helper/Nightmare.js]: Good point @PeterNgTr  :) 
Can we consider `for of` ? I think it is more readable. Your view?

Comment (PeterNgTr) [reply lib/helper/Nightmare.js]: should be good :)

**Comment by koushikmohan1996** [comment]
@DavertMik Can you merge this?

**Comment by koushikmohan1996** [comment]
@DavertMik Is it necessary to add this to 3.0? This is just an issue fix and it is not a breaking change.

**Pull request merged**